docs/getting-started/integrating-codacy-with-intellij-ides.md

```diff
@@ -0,0 +1,92 @@
+# Integrating Codacy with IntelliJ IDEs
```

Comment:
Closing this one, since we discussed this during the review.

For future reference:
* [IntelliJ IDEA](https://www.jetbrains.com/idea/) is is an IDE
* IntelliJ produces [a range of IDEs and other products](https://www.jetbrains.com/products/), including IDEA. The almost identical wording is an unfortunate coincidence.